.github/workflows/dockerfile-to-ecr.yaml

```diff
@@ -101,15 +101,15 @@ jobs:
         docker push $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG
         echo "image=$ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG" >> $GITHUB_OUTPUT
 
-    - name: Build, tag, and push patient-search image to Amazon ECR
-      id: patient-seatch
+    - name: Build, tag, and push nbs-modernization image to Amazon ECR
+      id: modernization-api
       env:        
         ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
         ECR_REPOSITORY: ${{ secrets.ECR_REPO_NAME }}
-        IMAGE_TAG: "${{ env.github_repo_name }}-patient-search-${{ env.github_branch }}-${{ env.github_sha_short }}"
+        IMAGE_TAG: "${{ env.github_repo_name }}-modernization-api-${{ env.github_branch }}-${{ env.github_sha_short }}"
       run: |
         # Build a docker container and push it to ECR 
-        docker build -t $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG ./patient-search
+        docker build -t $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG ./apps/modernization-api
```

Comment:
Are you now splitting up the api and ui into 2 separate containers? That is what I am getting from the 2 dockerfiles. If that is the case we will need to account for that in these build files. This may be a good time to split these up (probably an A&I task). Will require additional work for deployment to k8s as well.